specification/apicenter/resource-manager/Microsoft.ApiCenter/stable/2024-03-01/apicenter.json

```
@@ -0,0 +1,3088 @@
+        {
+          "$ref": "#/parameters/ApiDefinitionNameParameter"
+        },
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+        }
+      ],
+      "post": {
+        "tags": [
+          "ApiVersion"
+        ],
+        "summary": "Import API specification",
+        "operationId": "ApiDefinitions_ImportSpecification",
+        "description": "Imports the API specification.",
+        "parameters": [
+          {
+            "name": "payload",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ApiSpecImportRequest"
+            },
+            "description": "The API specification source entity."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The API specification was successfully imported."
+          },
+          "202": {
+            "description": "Request to import an API specification was accepted. The 'Location' response header contains the URL where the status of the long running operation can be checked."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ApiDefinitions_ImportSpecification": {
+            "$ref": "./examples/ApiDefinitions_ImportSpecification.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiCenter/services/{serviceName}/workspaces/{workspaceName}/apis/{apiName}/versions/{versionName}/definitions/{definitionName}/exportSpecification": {
+      "parameters": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+        },
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+        },
+        {
+          "$ref": "#/parameters/ServiceNameParameter"
+        },
+        {
+          "$ref": "#/parameters/WorkspaceNameParameter"
+        },
+        {
+          "$ref": "#/parameters/ApiNameParameter"
+        },
+        {
+          "$ref": "#/parameters/ApiVersionNameParameter"
+        },
+        {
+          "$ref": "#/parameters/ApiDefinitionNameParameter"
+        },
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+        }
+      ],
+      "post": {
+        "tags": [
+          "ApiVersion"
+        ],
+        "summary": "Export API specification",
+        "operationId": "ApiDefinitions_ExportSpecification",
+        "description": "Exports the API specification.",
+        "responses": {
```

Here is a workaround for the lintdiff error:
```suggestion
        "parameters": [],
        "responses": {
```

---

@mikekistler, are you suggesting replacing existing array of parameters with an empty array?